mmv1/products/redis/Instance.yaml

```
@@ -423,9 +423,9 @@ properties:
   - !ruby/object:Api::Type::String
     name: redisVersion
     description: |
-      The version of Redis software. If not provided, latest supported
-      version will be used. Please check the API documentation linked
-      at the top for the latest valid values.
+      The version of Redis software. If not provided, the current recommended
+      version will be used for creation. For updates, a not provided version has no effect.
```

```suggestion
      version will be used for creation. When updating, using a version that
      is not provided will have no effect.
```
is this an accurate rewording?

---

I am not a native English speaker, but I would assume the rewording sounds more like a version that is not "available". On the other hand, this might be a general inaccuracy with the docs - maybe it would be clearer to replace usages of _provided_ with _set_?

---

@AndreasBergmeier6176 I see, I think I misunderstood the wording myself. I think you are right, _set_ is clear.